ytfzf

```diff
@@ -982,6 +990,17 @@ user_selection () {
 	unset videos_data_clean
 }
 
+selected_data_from_file () {
+    selected_data="$( cat "$*" )"
+    [ -z "$selected_data" ] && exit
+	format_user_selection
+	get_video_format
+	get_sub_lang
+	play_url
+	save_before_exit
+    exit
+}
+
```

Comment:
instead of this maybe we just make this function set `selected_data` to the data in the file, then continue the main loop like normal (but skipping the `user_selection` bit), i'm not exactly sure what the best way to check to see if `user_selection` should be skipped though.

IMO, this function just feels very repetitive and inefficient, that's what I'm trying to solve.